libs/wingsdk/src/cloud/bucket.ts

```diff
@@ -345,6 +346,27 @@ export interface IBucketClient {
    * @inflight
    */
   publicUrl(key: string): Promise<string>;
+
+  /**
+   * Get the metadata of an object in the bucket.
```

Comment:
```suggestion
   * Get the metadata of an object in the bucket. Throws if there is no object with the given key.
```

Reply:
I've addressed that in https://github.com/winglang/wing/pull/4338/commits/5cfe0cc417cfff1adea53e4dde4ce4ce1d4d06df
`@throws` annotation is used.